components/video-list/video-list-non-featured.view.html

```diff
@@ -0,0 +1,12 @@
+<div infinite-scroll="infinite()" infinite-scroll-distance="1" infinite-scroll-disabled="loading">
+    <section class="{{sectionClass}}">
```

Comment:
remove some watchers by just adding :: to the name of the variable like this: {{::sectionClass}}